app/app.go

```diff
@@ -60,6 +61,12 @@ func (app app) runWorkflow(workflow w.Workflow, payload map[string]string) error
 	switch workflow.Identifier {
 	default:
 		runner = app.engine
+	case "afk_notifier":
+		matrixClient, err := app.botRegistry.GetClient("afk")
```

Comment:
Was this identifier where we agreed to use bot usernames?

Reply:
Yep, indeed, I opened https://github.com/Automattic/neurobot/issues/73 to track this, and will take care of it swiftly.